pkg/lib/oauth/response_mode.go

```diff
@@ -21,6 +21,7 @@ const htmlRedirectTemplateString = `<!DOCTYPE html>
 {{- else }}
 <script>
 {{- end }}
+window.parent.postMessage("{{ .redirect_uri }}", "{{ .custom_ui_uri }}")
```

Comment:
Also, I'd prefer not posting a raw string as a message, but an object for better flexibility. e.g. `{ redirect_uri: "..." }`